dask_geopandas/core.py

```diff
@@ -105,6 +105,17 @@ def spatial_partitions(self, value):
                 )
         self._spatial_partitions = value
 
+    @property
+    def _args(self):
+        # Ensure we roundtrip through pickle correctly
+        # https://github.com/geopandas/dask-geopandas/issues/237
+        return super()._args + (self.spatial_partitions,)
+
+    def __setstate__(self, state):
+        *dask_state, spatial_partitions = state
+        super().__setstate__(dask_state)
```

Comment:
Initially, I just copied over the `__setstate__` from dask.dataframe and added our state. By doing it like this (calling into `super()`) dask_geopandas should continue to work if Dask adds additional attributes to its state.